sentry/src/main/java/io/sentry/exception/InvalidDsnException.java

```diff
@@ -1,29 +1,32 @@
 package io.sentry.exception;
 
+import org.jetbrains.annotations.Nullable;
+
 public final class InvalidDsnException extends RuntimeException {
   private static final long serialVersionUID = 412945154259913013L;
-  private final String dsn;
+  private final @Nullable String dsn;
 
-  public InvalidDsnException(String dsn) {
+  public InvalidDsnException(final @Nullable String dsn) {
```

Comment:
Do we currently have a use case for this to be Nullable?
Ideally it would be non nullable.

Or best yet since this is a major bump: Get rid of this exception and save up on some code. Simplifies things with `new RuntimeException("Invalid DSN: " + dsn)`

@marandaneto wdyt?

it's not like someone is expecting InvalidDsnException to handle it somehow, are we?

Reply:
I'd go for `IllegalArgumentException` instead though

Reply:
`InvalidDsnException` removed.